README.md

```diff
@@ -0,0 +1,20 @@
+# example-sentence-extraction
+Module: Finding example sentences for keywords
+This module is responsible for fetching sentences containing the target user-inputed keyword, and by comparison with one another, select no more than 5 sentences which demonstrate the appropriate usage of the keyword.
+
+## Functional Design:
+External Package: MetaPy, NLTK
+- driver function loading json file into a dictionary
+![Image of load_json](https://github.com/Forward-UIUC-2021F/example-sentence-extraction/blob/main/load_json.png)
+
+- getSentences takes the target keyword and a text segment(string type) and returns a list of sentences containing the keyword. 
```

Comment:
I think you should make this function return a ranked list of sentences along with their scores.